src/lib/chips/chip-list.ts

```diff
@@ -174,6 +187,9 @@ export class MatChipList implements MatFormFieldControl<any>, ControlValueAccess
     return this.empty ? null : 'listbox';
   }
 
+  /** An object used to control when error messages are shown. */
+  @Input() errorStateMatcher: ErrorStateMatcher;
```

Comment:
I wonder if it would be better (at some point, not in this CL) to make `[errorStateMatcher]` its own directive. That way we could also place it on a `<div>`, `<form>`, etc. to change it for all its children